src/index.js

```diff
@@ -134,7 +135,9 @@ export default function(options = {}) {
                 codeExportDefault = getExport(result.opts.from)
                 Object.keys(codeExportDefault).forEach(k => {
                   const camelCasedKey = dashesCamelCase(k)
-                  codeExportSparse += `export const ${camelCasedKey}=${JSON.stringify(codeExportDefault[k])};\n`
+                  if (!reserved.check(camelCasedKey)) {
+                    codeExportSparse += `export const ${camelCasedKey}=${JSON.stringify(codeExportDefault[k])};\n`
+                  }
```

Comment:
maybe output a warning when a reserved keyword is used as well? like `You are using a served a keyword "xxx" as className so it's not available in named exports`